nextflow.config

```diff
@@ -252,45 +259,153 @@ docker.registry      = 'quay.io'
+        [
+            name: 'Júlia Mir Pedrol',
+            affiliation: 'Centre for Genomic Regulation (CRG)',
+            email: 'mirp.julia@gmail.com',
+            github: 'https://github.com/mirpedrol',
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: 'https://orcid.org/0000-0001-6104-9260'
+        ],
+        [
+            name: ' Laurence Kuhlburger',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/LaurenceKuhl',
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Matthias Hörtenhuber',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/mashehu',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Alan Tracey',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/alan-tracey',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Jasmin Frangenberg',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/jasmezz',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Marta Sanvicente-García',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/msanvicente',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Mike Schaffer',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/mschaffer-incyte',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Oskar Wacker',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/WackerO',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Susanne Jodoin',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/SusiJo',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
```

Comment:
Hi @SusiJo ,
we have added all the contributors to the pipeline and you contributed to crisprseq. Could you please update your data here? 
Thanks!